tests/func/test_live.py

```diff
@@ -255,3 +256,26 @@ def test_dvc_generates_html_during_run(tmp_dir, dvc, mocker, live_stage):
     live_stage(summary=True, live="logs", code=script)
 
     assert show_spy.call_count == 2
+
+
+def test_dvclive_running_subir(tmp_dir, scm, dvc):
+
+    tmp_dir.gen("subdir/train.py", LIVE_SCRIPT)
```

Comment:
Instead of using stringy paths:
```suggestion
    (tmp_dir / "subdir").gen("train.py", LIVE_SCRIPT)
```

or,
```suggestion
    tmp_dir.gen({"subdir": {"train.py": LIVE_SCRIPT}})
```